src/flows/data_retention/user_snowplow_events_flow.py

```diff
@@ -0,0 +1,28 @@
+"""Delete all raw snowplow events for users who have requested to have their accounts deleted.
+
+ Reference: https://getpocket.atlassian.net/wiki/spaces/CP/pages/2703949851/Snowflake+Data+Deletion+and+Retention
+"""
+
+from prefect import Flow
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery
+from utils.flow import get_flow_name
+
+FLOW_NAME = get_flow_name(__file__)
+
+# Run first day of month at midnight UTC.
+SCHEDULE = Schedule(clocks=[CronClock("0 0 1 * *")])
```

Comment:
This timing should be OK, but worth calling out that the Drop Deleted User Account Data dbt job [runs](https://cloud.getdbt.com/#/accounts/4171/projects/7569/jobs/17774/settings/) at 12pm UTC on the 1st of the month and the last step is to truncate the `STG_ACCOUNT_DELETIONS` table.

What this means is that between when that job ends, and the next time we run our nightly workload, the `STG_ACCOUNT_DELETIONS` table is empty and we want to avoid running this flow during that time window.

Reply:
Sure, we could definitely go that route!